server/src/main/java/io/unitycatalog/server/persist/dao/ModelVersionInfoDAO.java

```diff
@@ -90,7 +89,7 @@ public ModelVersionInfo toModelVersionInfo() {
             .source(getSource())
             .version(getVersion())
             .status(ModelVersionStatus.valueOf(getStatus()))
-            .storageLocation(FileUtils.convertRelativePathToURI(url))
+            .storageLocation(url)
```

Comment:
So TD and Ramesh are onboard with my quickstart proposal of:

1) We have users pip install mlflow
2) Users can then use a builtin dataset inside sklearn (library dependency of mlflow) to train a toy model.
3) User can then register that toy model directly into the default catalog/schema.
4) Users can then re-load the model from UC OSS and run batch inference from the same mlflow session.

This can all be done from a notebook/code we provide with the single step of installing mlflow and will give users the whole flow for using UC OSS w/ mlflow.

Reply:
So when users interact with MLflow, they will have a relative local directory that they train the model into.  Whatever we set as the default `storage-root.model` will be used as the storage location by UC OSS.

MLflow will then just make REST API calls to UC OSS, which will respond with the absolute paths that the mlflow client should use for a file based artifact upload (just a copy).

Reply:
The absolute paths will be generated by UC OSS based on the default storage root, which can be relative for quickstart.  Will probably want it to be something lie `./ucroot` or something like that if we don't want to go the route of letting java create a temp directory as the root.

Reply:
Fine, everything is MANAGED for models. Since this path will be created by UC on the fly, we don't need to worry about relative paths.